PYME/LMVis/importTextDialog.py

```diff
@@ -31,22 +31,33 @@ def isnumber(s):
     except:
         return False
     
-
-
-class ImportTextDialog(wx.Dialog):
+class ImportDialog(wx.Dialog):
```

Comment:
Would be better to call this 'ColumnMappingDialog` or similar - `ImportDialog` is a bit too generic